src/graph/tuning.cc

```diff
@@ -117,24 +117,26 @@ ncclResult_t ncclTopoTuneModel(struct ncclComm* comm, int minCompCap, int maxCom
     // Skipping on SCCL algorithms here since SCCL tune the algorithm in the synthesizer.
 
     for (int a=0; a<NCCL_NUM_ALGORITHMS; a++) {
-      if (coll == ncclFuncAllToAll || a == NCCL_ALGO_SCCL) {
-        // SCCL algorithm has hooks for AllToAll, AllGather, AllReduce and ReduceScatter
```

Comment:
why are you taking out allreduce?

Reply:
Weird. I think git did not give me a merge conflict for these.